conf.d/python.d/ceph.conf

```diff
@@ -0,0 +1,5 @@
+update_every : 10
+local:
+  config_file: '/etc/ceph/ceph.conf'
+  keyring_file: '/etc/ceph/ceph.client.admin.keyring'
+  presentation_data: 'KB'
```

Comment:
1. I suggest to move` update_every : 10` to the module file (`.py`).
2. `keyring_file`. Does it world readable? Do we need to add netdata to `ceph` group or something?
3. `presentation_data`. Netdata has units scaling feature `KB-MB-GB`. I think data should be in KB always.

And please copy any module conf file and modify it for ceph. It should contain all the stuff.

Reply:
Hi @l2isbad , thanks for the tips/suggestions. I really appreciate it!
1. I will put this on module file.
2. You are right. Moreover, you need change keyring access to group users can be used this file (I add netdata user and change permission file to 640). I need document it in some place. First, I will document on config file, and add a new session on (https://github.com/firehol/netdata/blob/master/python.d/README.md) explain ceph.
3. I define this variable because the default value is KB. In test environment, my ceph cluster contains 8GB and the presantation data is huge number. In my production environment, my ceph cluster contains +100TB and was impractible the huge number that graph shows. If no one presentation_data is defined, for default, it show this data on KB. But I will test again in my production environment to grab screenshots.

Reply:
> I define this variable because the default value is KB. In test environment, my ceph cluster contains 8GB and the presantation data is huge number. In my production environment, my ceph cluster contains +100TB and was impractible the huge number that graph shows. If no one presentation_data is defined, for default, it show this data on KB. But I will test again in my production environment to grab screenshots.

> When set to Scale Units the values shown will dynamically be scaled (e.g. 1000 kilobits will be shown as 1 megabit). Netdata can auto-scale these original units: kilobits/s, kilobytes/s, KB/s, KB, MB, and GB. When set to Fixed Units all the values will be rendered using the original units maintained by the netdata server. 

So units should be `KB` not `usage(whatever)`

Reply:
I see. But, when I get data on KB, the graph does not convert to other unity (like MB, GB, TB). For example: I caught Total usage of cluster on KB and my cluster have some 32/33 GB. The image shows how data is presented:
![image](https://user-images.githubusercontent.com/5157983/34314493-eb2728d8-e752-11e7-8f33-0f21cea2d94d.png)

33.140.660 is not well presented data. I could present it on another scale (GB, for example: 32,9). This was the reason to I use this approach.

How do I set Scale Units? Do you have some example? Thank you for the help.

Reply:
I see units is `usage` which is wrong. 

`usage` => `KB`

And dimensions should be `free` and `used` (instead `Total Usage Free`) and chart type 'stacked' 

`line` => `stacked`

```python
'general': {
    'options': [None, 'Ceph General Options', 'KB', 'general', 'ceph.general', 'stacked'],
    'lines': []
```

Reply:
I replace ```usage``` => ```KB``` and works to me (Auto-scaling). Thank you. I am ending last issues to be fix.